### ISSUE TITLE
Can I get a sample code for perspective transforming an image using cimg?

### ISSUE THREAD

**Issue by leeyunhome** [GreycLab/CImg#384]
Hello,

Thankfully, I'm using cimg just fine.
Can I get a sample code for perspective transforming an image using cimg?

Thank you.

**Comment by dtschump** [comment]
No, but you may ask ChatGPT, it seems it's able to do a pretty decent job at doing this.
